onnx2kerastl/constant_layers.py

```diff
@@ -19,4 +20,15 @@ def convert_constant_of_shape(node, params, layers, lambda_func, node_name, kera
     value = params.get('value')
     if value is None:
         raise NotImplementedError("ConstantOfShape should have a value param")
-    layers[node_name] = np.ones(layers[node.input[0]], dtype=np.int32)*params['value']
\ No newline at end of file
+    layers[node_name] = np.ones(layers[node.input[0]], dtype=np.int32)*params['value']
+
+
+def convert_one_hot(node, params, layers, lambda_func, node_name, keras_name):
+    axis = params.get('axis', -1)
+    layers[node_name] = tf.one_hot(indices=tf.cast(layers[node.input[0]],
+                                                   tf.int64),
```

Comment:
sure `tf.int64` and not `tf.int32`? usually tensorflow params are in float32 or int32

Reply:
This actually does not effect the output type and is one of the valid types of input. I put it there to align with how Onnx expects it.